changelog.d/7363.misc

```diff
@@ -0,0 +1 @@
+Async/await `RegistrationWorkerStore.is_server_admin`.
```

Comment:
Would "Convert `RegistrationWorkerStore.is_server_admin` and dependent code to async/await." be clearer?